.idea/GitAssemblyUtil.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I assume these files are IDE files? It's generally bad practice to check those in. The one exception I've come across is `.classpath` entries as those are a pain to set up every time you do a checkout.

Reply:
Deleted the .idea folder.